plugins/mcreator-localization/help/fr_FR/rangeditem/bullet_ignite_fire.md

```diff
@@ -1,3 +1,3 @@
-Check this parameter if you want it to light a fire when the ranged item hits a block.
+Cochez ce paramètre si vous voulez qu'il allume un feu lorsque l'objet à distance frappe un bloc.

```

Comment:
This translation is wrong... you should put "Cochez ce paramètre si vous voulez démarrer un feu sur le bloc touché par une balle de votre arme à distance."

Reply:
Problem still not fixed